platforms/org.eclipse.mita.platform.x86/src/org/eclipse/mita/platform/x86/platform/EventLoopGenerator.xtend

```diff
@@ -73,11 +73,11 @@ class EventLoopGenerator implements IPlatformEventLoopGenerator {
 	}
 	
 	override generateSetupPreamble(CompilationContext context) {
-		val startupEventHandler = context.allEventHandlers.filter[it.event instanceof SystemEventSource].findFirst[(it.event as SystemEventSource).source.name == "startup"]
+		val startupEventHandlers = context.allEventHandlers.filter[it.event instanceof SystemEventSource].filter[(it.event as SystemEventSource).source.name == "startup"]
```

Comment:
do resources other than the platform allow startup handler, too?

Reply:
you might have more than one startup handler